airflow-core/src/airflow/dag_processing/processor.py

```
@@ -200,20 +201,24 @@ def _parse_file_entrypoint():
         sys.path.append(bundle_root)
 
     result = _parse_file(msg, log)
+
     if result is not None:
         comms_decoder.send(result)
 
 
 def _parse_file(msg: DagFileParseRequest, log: FilteringBoundLogger) -> DagFileParsingResult | None:
     # TODO: Set known_pool names on DagBag!
 
+    warning_statement = check_dag_file_static(os.fspath(msg.file))
```

We should have a catch all exceptions here to ensure this doesn't break the parsing. We should manually test this implementation with very large dag file.